x-pack/legacy/plugins/siem/public/pages/hosts/index.tsx

```diff
@@ -153,7 +146,7 @@ export const HostsContainer = pure<HostComponentProps>(({ match }) => (
                 to={to}
                 setQuery={setQuery}
                 isInitializing={isInitializing}
-                {...props}
+                detailName={props.match.params.detailName}
```

Comment:
Note: Changed out the props spreading which includes the `match` object in favor of just the `detail` parameter name as that is a string and will not cause additional rendering of the React.memo.